pkg/config/config_test.go

```diff
@@ -1724,6 +1726,28 @@ func TestPatchesApplyToAllObjects(t *testing.T) {
 	}
 }
 
+func GetConfig(fileName string) (cfg OperatorConfig) {
+	file, err := os.Open(fileName)
+	if err != nil {
+		panic(err)
+	}
+	defer file.Close()
+	decoder := utilyaml.NewYAMLOrJSONDecoder(file, 100)
+	err = decoder.Decode(&cfg)
+	if err != nil {
+		panic(err)
+	}
+	return
+}
+
+func TestGraphDiffSanity(t *testing.T) {
+	proposedGraph := GetConfig("../../proposed-update-graph.yaml")
+	validatedGraph := GetConfig("../../validated-update-graph.yaml")
+	require.NotPanics(t, func() {
+		_ = proposedGraph.UpdateGraph.Difference(&validatedGraph.UpdateGraph)
```

Comment:
does this return an error, if so, is there a scenario it does not panic but returns an error and we are ignoring it?